src/resources/addys/claimables/types.ts

```diff
@@ -28,20 +28,33 @@ interface DApp {
   colors: Colors;
 }
 
-type ClaimableType = 'transaction' | 'sponsored';
-
-export interface AddysClaimable {
+interface AddysBaseClaimable {
   name: string;
   unique_id: string;
-  type: ClaimableType;
+  type: string;
```

Comment:
was this loosened for a reason?

Reply:
yeah, i realized backend types are looser